.github/workflows/action.yml

```diff
@@ -31,12 +31,24 @@ jobs:
       - name: Check if build left artifacts
         run: git diff --exit-code
 
-  unit_test:
-    name: Unit tests
+  unit_tests_ubuntu:
+    name: Unit tests (Ubuntu)
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
-        with: {node-version: '12'}
+        with:
+          node-version: 12
       - run: npm ci
       - run: npm test
+
+  unit_tests_windows:
```

Comment:
We were actually not testing this on Windows, and this change helped find a bug in the tests 😄